sunpy/net/jsoc/jsoc.py

```diff
@@ -702,18 +685,19 @@ def _make_recordset(self, series, start_time='', end_time='', wavelength='',
         keys = []
         keyword_info = c.keys(series)
         for key, value in keyword.items():
-            if key in keyword_info:
-                keys.append(f"{key}{value['operator']}{value['value']}")
-            else:
+            if key not in keyword_info:
                 raise ValueError(f"Keyword: '{key}' is not supported by series: {series}")
+            # We have to ensure that any values that are strings are quoted
+            other_value = value['value'] if str(value['value']).isnumeric() else f"'{value['value']}'"
+            keys.append(f"{key}{value['operator']}{other_value}")
```

Comment:
This is the bugfix that is the actual focus of this PR right?

Reply:
I'm also confused by this line. If as the comment says you are checking if this is a string or not, why not just do `if isinstance(value['value], str)`?

Reply:
Err, I forget now

Reply:
None of the unit tests failed for me locally when I did that, so I have no idea why I did that in the first place or the worrying one is that the unit tests don't cover this and I have broken something.

Only time and God can tell us.